pkg/integration/trivy/trivy.go

```diff
@@ -30,25 +31,17 @@ import (
 )
 
 var (
-	Repo          = getEnv("TRIVY_REPO", "https://aquasecurity.github.io/helm-charts/")
-	Version       = getEnv("TRIVY_VERSION", "0.13.0")
-	ChartName     = getEnv("TRIVY_CHART_NAME", "trivy-operator")
-	RepoShortName = getEnv("TRIVY_REPO_SHORT_NAME", "aqua")
-	ReleaseName   = getEnv("TRIVY_RELEASE_NAME", "trivy-operator-k8sgpt")
+	Repo          = cmp.Or("TRIVY_REPO", "https://aquasecurity.github.io/helm-charts/")
```

Comment:
I think that doesn't look right,
 the desired behavior is to see if an env variable is populated.

cmp.Or [function](https://pkg.go.dev/cmp#Or) in this implementation will make Repo's value to `TRIVY_REPO` string

I also don't see the value of using cmp to replace a small function

Reply:
got it